### ISSUE TITLE
Experiment details page says "Discuss title" instead of experiment title

### ISSUE THREAD

**Issue by pdehaan** [mozilla/testpilot#2058]
Found in https://testpilot.dev.mozaws.net/__version__ (4a96271b2e0633ac6031a193c8cb15ea8421a20a):

```json
{ "commit": "4a96271b2e0633ac6031a193c8cb15ea8421a20a",
  "version": "",
  "source": "https://github.com/mozilla/testpilot",
  "build": "https://circleci.com/gh/mozilla/testpilot/2758" }
```

---

Not sure if this is a regression, or if I'm just confused... Currently it looks like I see a label of "Discuss title":

![firefox_test_pilot_-_tab_center](https://cloud.githubusercontent.com/assets/557895/22274468/92b5069a-e25b-11e6-966a-44bba493995f.png)

It feels like something may not be getting parsed correctly and instead of "title" it should be the experiment title, like "Discuss Tab Center".

```html
<tr>
  <td data-l10n-id="discussExperiment" data-l10n-args={JSON.stringify({ title })}>Discuss {title}</td>
  <td><a href={discourse_url}>{discourse_url}</a></td>
</tr>
```

&mdash; Ref: [./frontend/src/app/containers/ExperimentPage.js:355-358](https://github.com/mozilla/testpilot/blob/4a96271b2e0633ac6031a193c8cb15ea8421a20a/frontend/src/app/containers/ExperimentPage.js#L355-L358).

<img width="714" alt="firefox_test_pilot_-_tab_center" src="https://cloud.githubusercontent.com/assets/557895/22274537/eddc265c-e25b-11e6-84e8-b092a68dc049.png">

**Comment by fzzzy** [comment]
It's supposed to be "Discuss {$title}" in the .ftl files, not "Discuss {title}". This bit me before

**Comment by fzzzy** [comment]
Also, I think it's best practice to leave the actual dom nodes blank instead of duplicating the english string. I suggest we slowly remove all the placeholder text from the dom nodes as portions of the code are touched, or do it all in one commit at some point.

**Comment by dannycoates** [comment]
fixed in #2059
